src/base/projects/View.svelte

```diff
@@ -45,10 +46,11 @@
 
     // Checks for delegates returned from seed node, as feature check of the seed node
     if (info.meta.delegates) {
-      const branches = await proj.getBranchesByPeer(urn, info.meta.delegates[0], cfg);
-      return { project: info, branches: [...Object.entries(branches.heads)], peer: info.meta.delegates[0], config: cfg, profile };
+      const branches = await proj.getBranchesByPeer(urn, peer || info.meta.delegates[0], cfg);
```

Comment:
This is still not right: we should be fetching branches of the selected peer, and only allowing the default branch when no peer is selected.